src/common/test/testUtils.tsx

```diff
@@ -21,6 +21,18 @@ export const mountWithApolloProvider = (
       {children}
     </MockedProvider>
   );
+
+export const mountWithApolloAndReduxProviders = (
```

Comment:
By default, I would have added the `<Provider>` into the current extended `mount` operation. In the long term it wouldn't make sense to have a new flavour whenever a new provider needs to be added.

If you wish, you can also consider the pattern shown here:

https://enzymejs.github.io/enzyme/docs/api/ReactWrapper/getWrappingComponent.html#examples

It would allow you to separate and share the `Provider` logic.

Reply:
I decided just to compine all provers under single helper. I tried to logic described in the docs, but it resulted to apollo network error.